source/numerics/vector_tools_project.inst.in

```diff
@@ -67,3 +67,40 @@ for (VEC : SERIAL_VECTORS ; deal_II_dimension : DIMENSIONS; deal_II_space_dimens
     \}
 #endif
 }
+
+for (deal_II_dimension : DIMENSIONS)
```

Comment:
@tamiko @kronbichler :    anyone knows how to add another list + parameter for instantiation in place, something like ` degree : {1; 2; 3; 4; 5}`? I tried a few things but was not successful as I don't really know who processes those `.inst.in` files, probably CMake.